addons/web/static/src/search/search_panel/search_view.scss

```diff
@@ -20,6 +20,13 @@
         background: inherit;
     }
 
+    .o_searchview_facet {
+        cursor: default;
```

Comment:
Unless I'm missing something this SCSS is not be needed.

`cursor: default` it's already "default" for `text-bg-action` facets,  and facets with domain have their :hover style managed by the button's classes. It's subtle but noticeable, especially on click.
Moreover it's consistent with the rest of the UI and adaps for dark-mode automatically

Reply:
for groupBy facets and several groupbys selected, the style of the cursor varies by moving over the facet

Reply:
I'm not sure to understand the issue. 16.3 seems to behave correctly. 
`cursor: pointer`  is applied on the facet you can interact with, the rest is default

https://github.com/odoo/odoo/assets/8558351/4dde6432-efa6-44ea-bb1a-0b6f5f3f6f4e

Reply:
ok